agate-rest/src/main/java/org/obiba/agate/web/rest/user/UserProfileResource.java

```diff
@@ -43,6 +43,4 @@
     if (user == null) NoSuchUserException.withName(username);
     return dtos.asDto(user, true);
   }
-
```

Comment:
Here I would expect the entry point for getting the translations of the user profile fields.